www-apache/modsecurity-crs/modsecurity-crs-3.3.4.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+DESCRIPTION="OWASP ModSecurity Core Rule Set"
+HOMEPAGE="https://coreruleset.org/"
+SRC_URI="https://github.com/coreruleset/coreruleset/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="amd64 x86"
+
+RDEPEND="|| ( >=www-apache/mod_security-2.9.6 >=dev-libs/modsecurity-3.0.8 )"
+
+S="${WORKDIR}/coreruleset-${PV}"
+
+src_install() {
+	insinto "/usr/share/${PN}"
+	doins -r rules util
+
+	dodoc CHANGES CONTRIBUTING.md CONTRIBUTORS.md KNOWN_BUGS README.md crs-setup.conf.example
+
+	if has_version www-servers/apache; then
```

Comment:
Alternatively we can add apache2/nginx USE flags so users can choose what to install. This is done in https://github.com/gentoo/gentoo/pull/27886 where apache2 is the default (to ease earlier apache2 installations), but nginx users can disable dependency on www-apache/mod_security by settings USE="-apache2 nginx".

Reply:
Yes. Using the flags is better as you suggested. I've updated accordingly.